.azure-pipelines/ci-scan.yml

```diff
@@ -0,0 +1,25 @@
+pool:
+  name: Hosted VS2017
```

Comment:
can it run on hosted ubuntu? build/package pipelines runs on ubuntu

Reply:
I don't think that will matter, as credscan & TSLint just scans the code and doesn't build it.